### ISSUE TITLE
Using with webpack

### ISSUE THREAD

**Issue by matepek** [paulmillr/chokidar#986]
**Describe the bug**

I'm trying to write a vscode extension and making it more compact I'm using webpack.
The webpack compiler fails. I'm not familiar with chockidar or webpack that much, I'm just looking for help.

```
ERROR in ./node_modules/fsevents/fsevents.node 1:0
Module parse failed: Unexpected character '�' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
(Source code omitted for this binary file)
 @ ./node_modules/fsevents/fsevents.js 13:15-41
 @ ./node_modules/chokidar/lib/fsevents-handler.js
 @ ./node_modules/chokidar/index.js
```

**Versions (please complete the following information):**
 - Chokidar version 3.3.1
 - Node version v13.10.1
 - OS version: MacOS 10.15.3
 - webpack: 4.41.6
 - ts-loader: 6.2.1

**To Reproduce:**

```
git clone https://github.com/matepek/vscode-catch2-test-adapter.git
git checkout chokidar
npm install
npm run webpack
```

See the error.

Also, I'm not sure that chokidar is platform independent for my case. If I create a package will that ork on other OS too?

**Comment by paulmillr** [comment]
`node_modules/fsevents/fsevents.node` is not a JS file. You or webpack are trying to include it. That seems like a webpack bug or misconfiguration.

**Comment by matepek** [comment]
I got that far. I also can ignore it manually. But would that work well for me?

Any reaction to my other question?

**Comment by paulmillr** [comment]
Yes, chokidar is platform-independent for your case. Don’t worry. Many folks are using it in electron apps.

**Comment by matepek** [comment]
So let's say I exclude that `fsevents.node` file which cause the problem. Will the lib download/build it again for the actual platform?

**Comment by paulmillr** [comment]
Removing fsevents would fuck up performance on macos. It wouldn't affect other platforms.

**Comment by repetier** [comment]
I suffer the same problem since chokidar version 3.2.3, so I compared the changes. It turned out that in 3.2.3 in package.json the line
`  "main": "index.js",`
was added. Nothing wrong with that I'd say and default value is even "index.js". None the less removing exactly this line makes the error disappear.
Can anyone make sense of this? I'm the only one compiling it so I can do that, but if that helps in other cases as well it might be worth investigating or just removing it again from package.json as it is not required anyway.
